src/syft_rds/client/rds_client.py

```diff
@@ -230,6 +305,7 @@ def _run(
         else:
             raise ValueError(f"Unknown display type: {display_type}")
 
-        runner = DockerRunner(handlers=[FileOutputHandler(), display_handler])
-        return_code = runner.run(config)
-        return return_code
+        runner = SyftRunner(
```

Comment:
Its unclear that SyftRunner is actually a factory class, and not a class. Can we switch it to some more factory naming? for example:

```
runner_cls = get_runner_cls(config)
runner = runner_cls(andlers=[FileOutputHandler(), display_handler], ...)
```